web/src/components/faq.js

```diff
@@ -76,7 +75,7 @@ const QuestionAnswer = ({
       <div
         className="answer"
         dangerouslySetInnerHTML={{
-          __html: co2Sub(__(`${groupKey}.${entryKey}-answer`)),
+          __html: __(`${groupKey}.${entryKey}-answer`),
```

Comment:
no need for dangerouslySetInnerHTML?

Reply:
I wasn't so confident about removing it in the FAQ section as different translations might contain links (and afaik some of them do).